src/utils/MultiInviter.ts

```diff
@@ -170,6 +172,32 @@ export default class MultiInviter {
                     errcode: USER_ALREADY_INVITED,
                     error: "Member already invited",
                 });
+            } else if (member?.membership === "ban") {
+                let proceed = false;
+                const ourMember = room.getMember(this.matrixClient.getSafeUserId());
+                if (
+                    !!ourMember &&
+                    member.powerLevel < ourMember.powerLevel &&
+                    room.currentState.hasSufficientPowerLevelFor("ban", ourMember.powerLevel) &&
+                    room.currentState.hasSufficientPowerLevelFor("kick", ourMember.powerLevel)
+                ) {
```

Comment:
it might have been nice to pull this out to a `canUnbanUser(ourMember, member)` function, but whatever

Reply:
Ideally it'd have been a util in the js-sdk but 🤷